news/wrappers_len.rst

```diff
@@ -0,0 +1,6 @@
+New features
+------------
+
+* __len__ function for AstypeWrapper
+* __len__ function for AsStrWrapper
+* __len__ function for FieldsWrapper
```

Comment:
The class names aren't really obvious in the public API - I'd probably describe them as `dset.astype()` etc.